Examples/outputSinDAC.py

```diff
@@ -1,3 +1,8 @@
+from __future__ import division
+from __future__ import print_function
```

Comment:
Minimum required version has been changed to 2.6 in setup.py. 
I will be pushing another commit to remove the things used to support 2.5.